amp/WEB-INF/src/org/digijava/module/aim/action/AuditLoggerManager.java

```diff
@@ -61,11 +65,30 @@ public ActionForward modePrepare(ActionMapping mapping, ActionForm form,
             }
                 
         }
+
         Collection<AmpAuditLogger> logs=AuditLoggerUtil.getLogObjects(vForm.isWithLogin());
         
+        if (vForm.getSelectedUser() != null || vForm.getFilteredTeam() !=null) {
+            if(vForm.getSelectedUser() != -1 && vForm.getFilteredTeam().equals("-1")){ 
+                logs = AuditLoggerUtil.getFilteredUser(vForm.isWithLogin(),vForm.getSelectedUser());
+            }
+            else if(!vForm.getFilteredTeam().equals("-1") && vForm.getSelectedUser() == -1 ){
+                logs = AuditLoggerUtil.getFilteredTeam(vForm.isWithLogin(),vForm.getFilteredTeam());
+          }
+            else if(!vForm.getFilteredTeam().equals("-1") && vForm.getSelectedUser() != -1 ) {
+                logs = AuditLoggerUtil.getFilteredUserAndTeam(vForm.isWithLogin(),vForm.getSelectedUser(),vForm.getFilteredTeam()); 
+            }
+            else if(vForm.getDateFrom() != null && vForm.getDateTo() != null) {
```

Comment:
I wouldn't require the user to enter both since you might want to be able to select from a certain date, to a certain date. between certain date.

i would send both date to getFilterByDate if one of them is !=null and let that method to filter using the values if !=null